pkgs/development/embedded/blisp/default.nix

```diff
@@ -5,17 +5,22 @@
 , cmake
 , libserialport
 , pkg-config
+, testers
 , IOKit
+, blisp
 }:
 
+let
+  version = "0.0.4";
```

Comment:
You dont need to introduce a let binding for this. Just change the version down below and use the finalAttrs pattern for stdenv.

Reply:
Thank you.
IIUC, this way we also don't break the override functionality.

Reply:
> IIUC, this way we also don't break the override functionality.

correct.